packages/common/src/messages/walletMessages.ts

```diff
@@ -83,5 +83,8 @@ export const walletMessages = {
   receiveTokensLearnMore: 'Learn More',
   receiveTokensCopy: 'Copy Wallet Address',
   receiveTokensClose: 'Close',
-  receiveTokensCopied: 'Copied to Clipboard!'
+  receiveTokensCopied: 'Copied to Clipboard!',
+  becomeMemberTitle: 'Become a member',
+  becomeMemberBody: (coinTicker: string = '') =>
```

Comment:
feel like empty string default is weird, would probably prefer to just not call this fn and return null until coinTicker is loaded in